autogluon/training/docker/0.7/py3/cu117/Dockerfile.gpu

```diff
@@ -0,0 +1,55 @@
+ARG CUDA_VER=11.7
+ARG PYTHON_VERSION=3.9.16
+
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:1.13.1-gpu-py39-cu117-ubuntu20.04-sagemaker
+
+# PYTORCH_SKIP_CUDNN_COMPATIBILITY_CHECK - see https://github.com/autogluon/autogluon/issues/2534
+ENV PYTORCH_SKIP_CUDNN_COMPATIBILITY_CHECK=1
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+RUN apt-get update \
+ && apt-get -y upgrade \
+ && apt-get install -y --no-install-recommends \
+ && apt-get autoremove -y \
+ # Install tesseract-ocr to support automm document classification
+ && apt-get install tesseract-ocr -y \
+ # Install rsync to support ray distributed training
+ && apt-get install rsync -y \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/* \
+ && rm -rf /tmp/*
+
+ARG AUTOGLUON_VERSION=0.7.0
+
+# NOTE: reinstalling horovod with pytorch support because it is available in the container
+RUN pip install --no-cache-dir -U --trusted-host pypi.org --trusted-host files.pythonhosted.org pip \
+ && pip install --no-cache-dir -U setuptools wheel \
+ && pip install --no-cache-dir -U numpy numba \
+ && pip install --no-cache-dir -U autogluon==${AUTOGLUON_VERSION} \
+ && mim install -q mmcv-full \
+ && pip install --no-cache-dir -U mmdet
+
+# Removing GluonTS nursery/tsbench package - it is not used in training/inference and have security vulnerabilities
+RUN rm -rf /usr/local/lib/python9/dist-packages/gluonts/nursery/tsbench
```

Comment:
`python9` -> `python3.9`?

Reply:
Good catch. I think it is a typo during copy paste